plot_continuous_monitor.py

```diff
@@ -77,16 +78,17 @@ def plot_continuous_monitor(filename, open_graphs=False):
     running_compute_monotasks = 0
     if "Running Compute Monotasks" in json_data:
       running_compute_monotasks = json_data["Running Compute Monotasks"]
+    xvdf_running_disk_monotasks = 0
+    xvdb_running_disk_monotasks = 0
+    if "Running Disk Monotasks" in json_data:
```

Comment:
That's for the branch master. I'm parsing logs created by the branch spark_with_logging:
https://github.com/NetSys/spark-monotasks/blob/spark_with_logging/core/src/main/scala/org/apache/spark/performance_logging/ContinuousMonitor.scala#L61